specification/ai/data-plane/Face/preview/v1.1-preview.1/Face.json

```diff
@@ -0,0 +1,7948 @@
+{
```

Comment:
I think you should populate the GA API in the "stable/v1.0" folder, to establish the baseline for tooling.

There are two ways to do this. One is to make your TypeSpec versioned, with a v1.0 version and a v1.1-preview version. I think this is the preferred solution. Or you could just copy the directory over from the cognitiveservices directory as is.

Reply:
I thought we only need to support latest version when migrating to TypeSpec. That's what I observed from document intelligence and image analysis. I'll go for copy the swagger file. Please let me know if you think we should include them in TypeSpec.

Reply:
I copied the stable swagger to current ai folder. How do I know whether it works for tooling? Would it take effect on APIView or it's other tool?

Reply:
I'm going to revert the swagger commit for now. Because Jeffery said he is going to check more detail about folder change of cognitive service. Copying swagger also bring more validation failure.